packages/time-utils/src/lib/Stopwatch.ts

```diff
@@ -0,0 +1,94 @@
+import { performance } from 'perf_hooks';
```

Comment:
Please make `@sapphire/stopwatch` because `perf_hooks` cannot be browerified so we cannot bundle this with Rollup